Enhanced footer (#525)
kitconcept/volto-light-theme: docs/how-to-guides/upgrade-guide.md

Although it is not strictly mandatory, it is recommended for a better experience that you install the new backend package `kitconcept.voltolighttheme` in your backend build.
Enable the site customization behaviors at least to your Plone site as shown in the {ref}`site-customization` documentation.

### New look and feel and features for the footer
```{versionadded} 6.0.0-alpha.22
```

The footer has been overhauled to be more modular and extensible without shadowing.
It has defined slots for the `preFooter`, `postFooter`, `footer`, `footerLinks`, and `followUs` areas.

It now also features the integration of `@plonegovbr/volto-social-media` so you can easily add social media links to your footer.

> See next section for more information.

Reviewer: ```suggestion
See the section {ref}`plonegovbr-volto-social-media-label` for more information.
```

> If you have customized the footer in your site, you might need to adapt your customizations to use the new elements and/or remove the things that your design doesn't need.

Reviewer: ```suggestion
If you have customized the footer in your site, you might need to adapt your customizations to use the new elements or remove the things that your design doesn't need.
```

```{image} /_static/footer6.png

> :alt: New footer in VLT 6.0.0

Reviewer: ```suggestion
:alt: Updated footer in VLT 6.0.0
```

```

> ### `@plonegovbr/volto-social-media` integration

Reviewer: ```suggestion

(plonegovbr-volto-social-media-label)=

### `@plonegovbr/volto-social-media` integration
```

```{versionadded} 6.0.0-alpha.22
```

> Not a breaking change, but a new feature worth mentioning since it needs a new behavior `plonegovbr.socialmedia.links` to be installed in your Plone site (or subsite, if you want to).

Reviewer: I don't think we need to mention that it's not a breaking change.

```suggestion
`@plonegovbr/volto-social-media` is a new feature that adds social media links and icons to your Plone site or subsite footer.
It requires a new behavior `plonegovbr.socialmedia.links` to be installed in your Plone site or subsite.
```

See the {ref}`social-media` section for more information.

> ### All feature flags has been removed

Reviewer: ```suggestion

### All feature flags have been removed
```
